src/Alert/Alert.tsx

```diff
@@ -21,16 +20,16 @@ export interface AlertProps {
   title: string;
 }
 
-function getIcon(variant: AlertVariant): string {
+function getIcon(variant: AlertVariant): React.ReactElement {
   switch (variant) {
     case "error":
-      return errorIcon;
+      return <NotAllowed />;
```

Comment:
I think each icon should have `icon` in its name, otherwise to be explicit about what element this is we'd have to rename imports each time. Also, look at line 5 - `import CloseIcon from "@material-ui/icons/Close";` - now we'd have some icons named with `Icon` and some not

Reply:
updated